tests/unit/GithubUtilsTest.js

```diff
@@ -25,11 +25,11 @@ describe('GithubUtils', () => {
                 },
             ],
             // eslint-disable-next-line max-len
-            body: '**Release Version:** `1.0.1-47`\r\n**Compare Changes:** https://github.com/Expensify/App/compare/production...staging\r\n\r\n**This release contains changes from the following pull requests:**\r\n- [ ] https://github.com/Expensify/App/pull/21\r\n- [x] https://github.com/Expensify/App/pull/22\r\n- [ ] https://github.com/Expensify/App/pull/23\r\n\r\n',
+            body: '**Release Version:** `1.0.1-47`\r\n**Compare Changes:** https://github.com/Expensify/App/compare/production...staging\r\n\r\n**This release contains changes from the following pull requests:**\r\nhttps://github.com/Expensify/App/pull/21\r\n        - [ ] QA\r\n        - [ ] Accessibility\r\n\r\nhttps://github.com/Expensify/App/pull/22\r\n        - [x] QA\r\n        - [ ] Accessibility\r\n\r\nhttps://github.com/Expensify/App/pull/23\r\n        - [ ] QA\r\n        - [ ] Accessibility\r\n\r\n',
```

Comment:
I don't see where the eight spaces before each checkbox are coming from. Is that intentional?